exporters/variables-scss/src/index.ts

```diff
@@ -1,21 +1,15 @@
 import { FileHelper } from '@supernovaio/export-helpers';
 import {
   AnyOutputFile,
-  ColorToken,
   PulsarContext,
   RemoteVersionIdentifier,
   Supernova,
-  TokenType,
+  OutputTextFile,
 } from '@supernovaio/sdk-exporters';
 import { ExporterConfiguration } from '../config';
-import { colorTokenToCSS } from './content/token';
-import { generateSimple } from './generators/simple';
+import { generateFiles } from './content/generators/fileGenerator';
 
-/**
- * Export entrypoint.
- * When running `export` through extensions or pipelines, this function will be called.
- * Context contains information about the design system and version that is currently being exported.
- */
+// @ts-ignore-next-line
```

Comment:
Please, add a "why" part of this comment.

Reply:
added link to https://github.com/Supernova-Studio/exporters/issues/4